### ISSUE TITLE
Vue1 incompatibility

### ISSUE THREAD

**Issue by cyberbobs** [noru/vue-easy-toast#7]
This component is currently not compatible with Vue 1.0.x. Since version 0.1.0 it uses `transition` element which is only available in Vue 2.x so when trying to show the toast we have this error in console:

`[Vue warn]: Unknown custom element: <transition> - did you register the component correctly? For recursive components, make sure to provide the "name" option.`

**Comment by noru** [comment]
@cyberbobs thanks for reporting. I noticed that warning, but that should not affect the functionality.

I tried to suppress the warning but didn't work. Let me try again.

**Comment by noru** [comment]
Hi @cyberbobs it should be no [Vue warn] in v0.2.0. Would you please help to verify it?
